test/remote/flow_tests.js

```diff
@@ -112,6 +112,6 @@ describe('remote flow', function() {
   )
 
   after(() => {
-    return TestServer.stop(server)
+    TestServer.stop(server)
```

Comment:
@deeptibaghel now that we downgraded `mocha`, do we still need to do this?

Reply:
Downgraded mocha? Were there problems with a newer version?

Reply:
@vladikoff yes it throws 
"after all" hook:
Error: timeout of 15000ms exceeded. Ensure the done() callback is being called in this test.
if we keep "return"
@shane-tomlinson yes the latest mocha version 5.2.0 causes the tests to hang at the end.